packages/styleguide/tsup.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'tsup'
+
+const config = defineConfig({
+  entryPoints: ['src/lib.ts', 'src/editor.ts', 'src/chart.ts'],
+  format: ['cjs', 'esm'],
+  dts: true,
+  sourcemap: true,
+  clean: true,
+  minify: true,
+  target: 'es6',
+  external: ['react', 'react-dom'],
```

Comment:
This should not be necessary. `dependencies` and `peerDependencies` are automatically excluded.